EVPurchaseProgram/src/main/java/ro/siit/evprogram/CarsFullPrice.java

```diff
@@ -0,0 +1,21 @@
+package ro.siit.evprogram;
+
+import java.util.ArrayList;
+
+public class CarsFullPrice {
+
+    /**
+     * Method for getting the car's full price
+     *
+     * @param cd
+     * @return the car's full price
+     */
+
+    public int getPrice(ArrayList<CarDealership> cd) {
```

Comment:
Why do you need a separate class for this? Why isn't it part of the CarDealership class?